### ISSUE TITLE
Door flag boolean option broken

### ISSUE THREAD

**Issue by tiggerbiggo** [Zet0rz/nzombies#600]
Hey, love this gamemode, however there's a bug that is stopping me from making maps properly. Whenever I either use the context menu or the tool to set a door to be unlocked, the "Enable Flag" option will never be set, even if it is ticked within the tool or context menu. This means that I can't use flags at all. The map is cs_office, and it means I can't have zombies spawn outside of the first room because i'd need to use flags. I've tried with multiple different props to no avail. Anyone know what's going on here? Thanks.

**Comment by DjNathann** [comment]
Are you trying to set flag using map props?

**Comment by tiggerbiggo** [comment]
@DjNathann  Yeah, the ones in the "Gates" section. I found out it actually does set the flags, but when you open the context menu to edit it, the tickbox isn't ticked even if it technically does enable the flag. Still an issue, just not quite as bad.

**Comment by minin43** [comment]
Yes, I've worked with this gamemode for nigh on 4 years (off and on so not straight, sue me), and that bug's pretty much always been around. Just because it's unchecked doesn't mean it actually isn't using the trigger. Mainly just a visual (per-se) bug.

**Comment by tiggerbiggo** [comment]
Gotcha, I guess this should be left here unless there's already an open issue about it. If anyone finds an issue for this same thing feel free to close this one.
